tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyLog.java

```diff
@@ -28,35 +22,28 @@
     static boolean DEBUG = false;
     private static boolean checkDebug = false;
     // Logging
-    public static Logger logger;
+    private static Logger logger;
 
     static {
-        logger = Logger.getLogger("org.unicode.cldr.SurveyMain");
-        // if(DEBUG) {
-        // logger.setLevel(Level.ALL);
-        // // for(Handler h : logger.getHandlers()) {
-        // // h.setLevel(logger.getLevel());
-        // // }
-        // ConsoleHandler ch = new ConsoleHandler();
-        // ch.setLevel(logger.getLevel());
-        // logger.addHandler(ch);
-        // }
+        logger = Logger.getLogger(SurveyLog.class.getName());
     }
 
-    public static java.util.logging.Handler loggingHandler = null;
-
+    @Deprecated
     static final void errln(String s) {
         logger.severe(s);
     }
 
+    @Deprecated
     public static void logException(Throwable t) {
         logException(t, "", null);
     }
 
+    @Deprecated
     public static void logException(Throwable t, String what) {
         logException(t, what, null);
     }
 
+    @Deprecated
```

Comment:
Sometimes deprecation warnings have the drawback of making more important warnings harder to spot. For example, currently Eclipse reports 3211 warnings. Who knows how many of them are actual bugs we're not fixing, and how many are just wishful thinking? It might be good policy if we only add deprecation in the context of an actual plan -- with a ticket and a due date -- for removing the deprecated items. Just my 2 cents worth

Reply:
it's a refactoring aid. Ticket completion would be removing all usages and deleting this method.

Reply:
In my way of thinking I would deprecate SurveyAjax  … the same way JSP is deprecated for you (and for me).  But I can see the value of keeping the error volume down. 

as to getrow, I'd change a callsite to use the new getrow rather than update the old one.